src/commands/commandConstants.ts

```diff
@@ -52,11 +52,18 @@ export namespace CommandConstants {
     export const OPEN_DOCS_HOME = "xml.open.docs.home";
 
     /**
-     * VSCode client command to executes an LSP command on the XML Language Server
+     * VSCode client commands to revalidate files with an LSP command on the XML Language Server

```

Comment:
Move this comment down to the line before `xml.validation.current.file`.

Reply:
fixed